src/test/datascience/notebook/helper.ts

```diff
@@ -344,7 +344,7 @@ export async function waitForKernelToGetAutoSelected(expectedLanguage?: string,
                 );
                 return preferred != undefined;
             },
-            3_000,
+            30_000,
```

Comment:
This timeout was failing because now remote connections try to match interpreters. Getting intepreter details was taking longer than 3 seconds.

Reply:
This is what was causing my test failures AFAIK. Hopefully next run completes fully.